ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -516,6 +516,16 @@ import '../nuxeo-button-styles.js';
       };
     }
 
+    _isChecked(selectAllActive, _excludedItems) {
+      if (selectAllActive && _excludedItems.length <= 0) {
+        return true;
+      }
+      if (selectAllActive && _excludedItems.length > 0) {
+        return false;
+      }
```

Comment:
Kindly recheck conditions Block 523 -525 is unnecessary.

Reply:
thankyou @rajatgoel-gl ,did the chnages.